string FileName
int32 Lap
RoiList[] Rois
bool TestVideo
string VideoPath
string DataPath
int32[] HSV_lower
int32[] HSV_upper